### PR TITLE
LWDEV-8709 Fix responses for buildingShouldBeRepeated error

### PR DESCRIPTION
**Pull request by cassade**
Added expiration height (in "closed ledgers") setting

### CODE REVIEW

File: core/index-api.js [hunk 1]
```diff
@@ -319,12 +319,9 @@ let findTx = async (ctx) => {
 			fee: (tx.fees || 0) + '',
 			hash: tx.hash,
 			block: tx.block || undefined,
-			error: tx.error || undefined
+			error: tx.error || undefined,
+			errorCode: tx.errorCode
```
Comment (KonstantinRyazantsev) [inline core/index-api.js:323]: don't we need tx.errorCode || "unknown" here?

File: core/index-api.js [hunk 2]
```diff
@@ -887,7 +888,8 @@ let API_ROUTES = {
 						}
 
 						await ctx.store.tx(tx._id, {
-							error: errorCode, 
+							errorCode: errorCode,
```
Comment (KonstantinRyazantsev) [inline core/index-api.js:891]: don't see errorCode definintion. Should it be tx.errorCode?

File: core/index-api.js (hunk 1 above)
Comment (cassade) [reply core/index-api.js:323]: Not really - it's nullable in contract.

File: core/index-api.js (hunk 2 above)
Comment (cassade) [reply core/index-api.js:891]: No, it should not. Definition was there, with this change I just fixed its using.

**Review by KonstantinRyazantsev — APPROVED** [review]

**Pull request merged**